source/UF/Replacement.lagda

```diff
@@ -0,0 +1,108 @@
+Ian Ray, 17th August 2025.
+
+The type theoretic axiom of replacement (section 24.5 of Introduction to
+Homotopy Type Theory by Egbert Rijke:
+https://ulrikbuchholtz.dk/hott1920/hott-intro.pdf)
+is a statement about the size of the image of a function when certain smallness
+assumptions are imposed on the domain and codomain. The nomenclature is derived
+from the set theoretic axiom of replacement, where 'sethood' provides a notion
+of size. In type theory, this statement may be assumed or proven depending on
+the context. Thus, we will call it 'type replacement' or simply 'replacement'
+when there is no risk of confusion with set replacement (see Size.lagda).
```

Comment:
Sorry but now the comment is more confusing than before as it currently suggests that `set` in `set replacement (see Size.lagda)` corresponds to *size* (set vs class), but it doesn't, it refers to a truncation level.